plugins/techdocs/src/reader/components/TechDocsHome.test.tsx

```diff
@@ -13,14 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { TechDocsHome } from './TechDocsHome';
```

Comment:
only automatic reordering, remnant from before i re-exported the client through the frontend plugin